### PR TITLE
OLM-Low-OCP-26044-catalog-operator memory and time consumes 11GB

### PR DESCRIPTION
**Pull request by tbuskey**
Test Case [OCP-26044](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-26044) & part of Jira [OLM-1428](https://issues.redhat.com/browse/OLM-1428)

Does the following:
- `oc get pods -n openshift-operator-lifecycle-manager` _to find the catalog-operator pod_
  - extract the full name of the pod
  - extract the memory requested from the pod
  - extract how long the pod has been up
- `oc adm top pod -n openshift-operator-lifecycle-manager` on the pod _to extract how much memory is in use_
- Add the info that has been gathered in the log
  - if there is a failure on the way, add all gathered info to the log before failing
  
#### How to test:
`./extended-platform-tests run all --dry-run | grep 'catalog-operator memory and time consumes' | ./extended-platform-tests run -f - `

> oc version
> Client Version: openshift-clients-4.4-51-g5d7a12f0
> Server Version: 4.4.0-0.nightly-2020-02-16-221315
> Kubernetes Version: v1.17.1

### CODE REVIEW

**Comment by tbuskey** [comment]
@jianzhangbjz  @bandrade  @scolange  @kuiwang02 @mffiedler

File: test/extended/operators/olm.go [hunk 1]
```diff
@@ -251,6 +251,72 @@ var _ = g.Describe("[Feature:Platform] an end user use OLM", func() {
 		}
 	})
 
+
+	// author: tbuskey@redhat.com
+	g.It("OLM-Low-OCP-26044-catalog-operator memory and time consumes", func() {
```
Comment (jianzhangbjz) [inline test/extended/operators/olm.go]: Should be `Medium`, not `Low`.

**Comment by tbuskey** [comment]
Move to private

**Pull request closed without merging**